pkg/operator/starter.go

```diff
@@ -383,21 +400,26 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	return nil
 }
 
-func apiServices() []*apiregistrationv1.APIService {
-	var apiServiceGroupVersions = []schema.GroupVersion{
-		// these are all the apigroups we manage
-		{Group: "apps.openshift.io", Version: "v1"},
-		{Group: "authorization.openshift.io", Version: "v1"},
-		{Group: "build.openshift.io", Version: "v1"},
-		{Group: "image.openshift.io", Version: "v1"},
-		{Group: "project.openshift.io", Version: "v1"},
-		{Group: "quota.openshift.io", Version: "v1"},
-		{Group: "route.openshift.io", Version: "v1"},
-		{Group: "security.openshift.io", Version: "v1"},
-		{Group: "template.openshift.io", Version: "v1"},
+func apiServices(clusterVersionInformer configinformersconfigv1.ClusterVersionInformer) ([]*apiregistrationv1.APIService, []*apiregistrationv1.APIService, error) {
+	clusterVersion, err := clusterVersionInformer.Lister().Get("version")
```

Comment:
I think that we should add `clusterVersionInformer` to `WithInformers` here https://github.com/openshift/library-go/blob/75c7d51fc4155264ba71551d22d0a1968b7bf989/pkg/operator/apiserver/controller/apiservice/apiservice_controller.go#L66 because 

```
// WithInformers is used to register event handlers and get the caches synchronized functions.
```

Reply:
and maybe we should change `GetAPIServicesToMangeFunc` to accept `clusterVersionInformer configinformersconfigv1.ClusterVersionInformer` ?

Reply:
alternative would be to add `WaitForCacheSync` to the `apiServices` function and somehow trigger the controller to call `apiServices` when `clusterVersionInformer` changes/get an event.

Reply:
does it make sense?

Reply:
> I think that we should add clusterVersionInformer to WithInformers here

https://github.com/openshift/library-go/pull/1562

Reply:
+1 for passing informers as the list of managed api services may have many different implementation which do not need clusterVersionInformer.

Reply:
ok, thanks.